e2e/suite1/specs/queryVariableCrud.spec.ts

```diff
@@ -1,74 +1,172 @@
 import { e2e } from '@grafana/e2e';
+import { Flows } from '@grafana/e2e/src/flows';
```

Comment:
Shouldn't Flows be exported from e2e? Not saying no to path imports, but wondering if consumer shouldn't have access to Flows.

Reply:
Great suggestion, I'll do that!

Reply:
Aaah, I forgot I should use `e2e.flows` instead.

Reply:
perfect!